README.md

```diff
@@ -414,6 +414,7 @@ meetingSession.audioVideo.subscribeToActiveSpeakerDetector(
 a video stream, etc. To view a video in your application, you must bind a tile to a `<video>` element.
 > - Make sure you bind a tile to the same video element until the tile is removed.
 > - A tile is created with a new tile ID when the same attendee restarts the video.
```

Comment:
in the case of a removeLocalTile, it will use a different video tile Id, correct?